A blog on podcast "Bringing TypeScript to Ember" (#1173)
ember-learn/ember-blog: content/typed-ember/the-ember-times-issue-201.md

> # Whiskey, Web and Whatnot with Chris Krycho🔊

Reviewer: ```suggestion
# [Whiskey, Web and Whatnot with Chris Krycho🔊](https://www.whiskeywebandwhatnot.fm/bringing-types-to-ember-with-chris-krycho/)
```

> Robert Wagner (@rwwagner90) and Chuck Carpenter (@chuckcarpenter) of Ship Shape sit down with Chris Krycho(@chriskrycho) for a podcast episode.Chris shares his initial work on getting TypeScript into Ember, how Typed-Ember got established, further on explaining its need and long term benefits.

Reviewer: ```suggestion
Robert Wagner (@rwwagner90) and Chuck Carpenter (@chuckcarpenter) of Ship Shape sit down with Chris Krycho(@chriskrycho) in this episode of [Whiskey, Web and Whatnot](https://www.whiskeywebandwhatnot.fm). Chris shares his initial work on getting TypeScript into Ember, how Typed-Ember got established, further on explaining the need and long term benefits.
```

> They also discuss about the enthusiasm of newbies to use TypeScript and the complexities that might arise, factors to be considered for deciding, if they need to use TypeScript. Chris suggests developers to learn Rust programming, also for developers having object oriented programming knowledge can work on functional programming and vice-versa, which helps in broadening their horizon. A generic tip is to be aware of the happenings in the eco-system, it helps a lot in career progression.

Reviewer: ```suggestion
They also discuss the enthusiasm of newbies to use TypeScript and the complexities that might arise. If you want to know factors to consider when deciding to use TypeScript, they discuss a few. Chris suggests developers to learn Rust programming because learning functional programming helps broadening your horizon.
```
